### ISSUE TITLE
tango.util.Convert

### ISSUE THREAD

**Issue by SiegeLord** [SiegeLord/Tango-D2#24]
I hit a dmd bug trying to get it working (http://d.puremagic.com/issues/show_bug.cgi?id=7418), either wait for it to get fix or find a workabout. Specifically the issue is with this method:

```
static if( is( T : const(char[]) ) )
        alias tango.text.convert.Utf.toString toString_;

    else static if( is( T : const(wchar[]) ) )
        alias tango.text.convert.Utf.toString16 toString_;

    else
        alias tango.text.convert.Utf.toString32 toString_;
}
```

**Comment by SiegeLord** [comment]
Try this workabout:

```
T[] toString_(T, C)(const(C)[] str)
{
    static if( is( T : const(char[]) ) )
        return tango.text.convert.Utf.toString(str);

    else static if( is( T : const(wchar[]) ) )
        return tango.text.convert.Utf.toString16(str);

    else
        return tango.text.convert.Utf.toString32(str);
}
```

Template bloat etc, but might be good enough to tide us over until that bug gets fixed...

**Comment by SiegeLord** [comment]
Fixed for the most part in 439ad40a131118932a8287d33ee501d9570ea64f.
